pkg/integration/components/runner.go

```diff
@@ -165,7 +172,7 @@ func getLazygitCommand(test *IntegrationTest, paths Paths, rootDir string, sandb
 		cmdObj.AddEnvVars(fmt.Sprintf("KEY_PRESS_DELAY=%d", keyPressDelay))
 	}
 
-	cmdObj.AddEnvVars("GIT_CONFIG_GLOBAL=/dev/null")
+	cmdObj.AddEnvVars(fmt.Sprintf("%s=%s", GIT_CONFIG_GLOBAL_ENV_VAR, globalGitConfigPath(rootDir)))
```

Comment:
we're now supplying the global config ourselves, allowing all file transport forms so that we can add a submodule via a path